inc/ruleright.class.php

```diff
@@ -151,6 +152,9 @@ function executeActions($output, $params) {
                         $continue                   = false;
                         $output_src["_stop_import"] = true;
                         break;
+					case "groups_id" :
+						$output["groups_id"] = $action->fields["value"];
```

Comment:
virtual fields (not part of the glpi_users table should be _ prefixed.

This also mean a single group can be add, which sounds not correct